auto_submit/lib/requests/update_revert_reviews.dart

```diff
@@ -0,0 +1,80 @@
+// Copyright 2022 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:auto_submit/exception/bigquery_exception.dart';
+import 'package:auto_submit/model/big_query_revert_request_record.dart';
+import 'package:auto_submit/service/bigquery.dart';
+import 'package:auto_submit/service/github_service.dart';
+import 'package:github/github.dart';
+import 'package:shelf/shelf.dart';
+
+import '../service/log.dart';
+import '../server/authenticated_request_handler.dart';
+
+class UpdateRevertReviews extends AuthenticatedRequestHandler {
+  const UpdateRevertReviews({
+    required super.config,
+    required super.cronAuthProvider,
+  });
+
+  @override
+  Future<Response> get() async {
+    BigqueryService bigqueryService = await config.createBigQueryService();
```

Comment:
https://dart-lang.github.io/linter/lints/prefer_final_fields.html

Reply:
Haha yeah I keep forgetting. I will add this to the analyzer options too.